pkgdb/src/registry/floxpkgs/flake.nix.in

```diff
@@ -0,0 +1,272 @@
+# -*- mode: nix; -*-
+# ============================================================================ #
+#
+# Wraps `nixpkgs' to patch the package set for traversal by `pkgdb scrape'.
+#
+# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+# Packages should NEVER be consumed/built from this flake!
```

Comment:
ignore this comment, there's no _real reason_ not to build from this other than "it's disposable and may be hard to reproduce later".